data/json/npcs/BGSS_talk.json

```diff
@@ -0,0 +1,922 @@
+    "responses": [
+      { "text": "What about your shelter?", "topic": "BGSS_PREPPER_1_LMOE" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_PREPPER_1_LMOE",
+    "type": "talk_topic",
+    "dynamic_line": "I still haven't made it there. Every time I've tried I've been headed off by the dead. Who knows, maybe someday.",
+    "//": "In the future this NPC should give you the map coordinates to the LMOE shelter if you persuade.",
+    "responses": [
+      { "text": "Could you tell me that story again?", "topic": "BGSS_PREPPER_1_STORY1" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_PREPPER_2_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "Oh, man. I thought I was ready. I had it all planned out. Bug out bags. Loaded guns. Maps of escape routes. Bunker in the back yard.",
+    "responses": [
+      { "text": "Sounds like it didn't work out.", "topic": "BGSS_PREPPER_2_STORY2" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_PREPPER_2_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "Depends on your definition. I'm alive, aren't I? When Hell itself came down from the skies and monsters started attacking the cities, I grabbed my stuff and crammed into the bunker. My surface cameras stayed online for days; I could see everything happening up there. I watched those things stride past. I still have nightmares about the way their bodies moved, like they broke the world just to be here. I had nothing better to do. I watched them rip up the cops and the military, watched the dead rise back up and start fighting the living. I watched the nice old lady down the street rip the head off my neighbour's dog. I saw a soldier's body twitch and grow into some kind of electrified hulk beast. I watched it all happen.",
+    "responses": [
+      { "text": "Why did you leave your bunker?", "topic": "BGSS_PREPPER_2_STORY3" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_PREPPER_2_STORY3",
+    "type": "talk_topic",
+    "dynamic_line": "Honestly? I was planning to die. After what I'd seen, I went a little crazy. I thought it was over for sure, I figured there was no point in fighting it. I thought I wouldn't last a minute out here, but I couldn't bring myself to end it down there. I headed out, planning to let the monsters finish me off, but what can I say? Survival instinct is a funny thing, and I killed the ones outside the bunker. I guess the adrenaline was what I needed. It's kept me going since then.",
+    "responses": [
+      { "text": "Why did you leave your bunker?", "topic": "BGSS_PREPPER_2_STORY3" },
```

Comment:
copy-paste error?  Otherwise, this is a dead loop.
Also, consider having an effect of u_buy_item: survivor_map so he can give you one his maps.

Reply:
The map is a cool idea! For a number of the NPCs that have specific place knowledge, I want to eventually be able to give the player a mission to go to that place, or at least the option to get map coords from the NPC. I know you're working on JSONifying missions, it can probably wait until then, because once those are JSON then the dialogue project is going to start overlapping with that a lot I suspect.